---
title: papers | Wen Huang Laboratory
layout: pages
permalink: /publications.html
active: publications
---

## papers

| | | |
|--|--:|--|
| <b>2020</b> | 48. | <b>Liu J</b>, <b>Tan S</b>, Huang S, <b>Huang W</b> (2020) ASlive: a database for alternative splicing atlas in livestock animals. <a href="https://www.ncbi.nlm.nih.gov/pubmed/32000661" target="blank"><i>BMC Genomics</i> 21(1):97 </a> |
| | 47. | Matute DR, Comeault AA, Earley E, Serrato-Capuchina A, Peede D, Monroy-Eklund A, <b>Huang W</b>, Jones CD, Mackay TFC, Coyne JA (2020) Rapid and predictable evolution of admixed populations between two Drosophila species pairs. <a href="https://www.ncbi.nlm.nih.gov/pubmed/31767631" target="blank"><i>Genetics</i> 214:211-230 </a> |
| | 46. | <b>Dai Z</b>, Long N, <b>Huang W</b> (2020) Influence of genetic interactions on polygenic prediction. <a href="https://www.ncbi.nlm.nih.gov/pubmed/31649046" target="blank"><i>G3</i> 10(1):109-115</a> |
| <b>2019</b>| 45. | Ding R, Yang M, Quan J, Li S, Zhuang Z, Zhou S, Zheng E, Hong L, Li Z, Cai G, <b>Huang W</b>, Wu Z, Yang J (2019) Single-locus and multi-locus genome-wide association studies for intramuscular fat in Duroc pigs. <a href="https://www.ncbi.nlm.nih.gov/pubmed/31316554" target="blank"><i>Front Genet</i> 10:619</a> |
| | 44. | Veturi Y, de Los Campos G, Yi N, <b>Huang W</b>, Vazquez AI, KÃ¼hnel B (2019) Modeling heterogeneity in the genetic architecture of ethnically diverse groups using random effect interaction models. <a href="https://www.ncbi.nlm.nih.gov/pubmed/30796011" target="blank"><i>Genetics</i> 211(4)1395-1407</a> |
| | 43. | Quan J, Cai G, Yang M, Zeng Z, Ding R, Wang X, Zhuang Z, Zhou S, Li S, Yang H, Li Z, Zheng E, <b>Huang W</b>, Yang J, Wu Z (2019) Exploring the fecal microbial composition and metagenomic functional capacities associated with feed efficiency in commercial DLY pigs. <a href="https://www.ncbi.nlm.nih.gov/pubmed/30761104" target="blank"><i>Front Microbiol</i> 10:52</a> |
| | 42. | Harbison ST, Kumar S, <b>Huang W</b>, McCoy LJ, Smith KR, Mackay TFC (2019) Genome-wide association study of circadian behavior in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/30341464" target="blank"><i>Behav Genet</i> 49(1):60-82</a> |
| <b>2018</b>| 41. | Ding R, Yang M, Wang X, Quan J, Zhuang Z, Zhou S, Li S, Xu Z, Zheng E, Cai G, Liu D, <b>Huang W</b>, Yang J, Wu Z (2018) Genetic architecture of feeding behavior and feed efficiency in a Duroc pig population. <a href="https://www.ncbi.nlm.nih.gov/pubmed/29971093" target="blank"><i>Front Genet</i> 9:220</a> |
| | 40. | Morgante F, <b>Huang W</b>, Maltecca C, Mackay TFC (2018) Effect of genetic architecture on the prediction accuracy of quantitative traits in samples of unrelated individuals. <a href="https://www.ncbi.nlm.nih.gov/pubmed/29426878" target="blank"><i>Heredity</i> 120(6):500-514</a> |
| | 39. | Mackay TFC, <b>Huang W</b> (2018) Charting the genotype-phenotype map: lessons from the <i>Drosophila melanogaster</i> Genetic Reference Panel. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28834395" target="blank"><i>WIRE Dev Biol</i> 7(1):e289</a> |
|<b>2017</b>| 38. | Fochler S, Morozova TV, Davis MR, Gearhart AW, <b>Huang W</b>, Mackay TFC, Anholt RRH (2017) Genetics of alcohol consumption in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/28627812" target="blank"><i>Genes Brain Behav</i> 16(7):675-685</a> |
| | 37. | Garlapow ME, Everett LJ, Zhou S, Gearhart AW, Fay KA, <b>Huang W</b>, Morozova TV, Arya GH, Turlapati L, St Armour G, Hussain YN, McAdams SE, Fochler S, Mackay TFC (2017) Genetic and genomic response to selection for food consumption in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27704301" target="blank"><i>Behav Genet</i> 47(2):227-243</a> |
|<b>2016</b> | 36. | <b>Huang W</b>, Mackay TFC (2016) The genetic architecture of quantitative traits cannot be inferred from variance component analysis. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27812106" target="_blank"><i>PLoS Genet</i> 12(11):e1006421</a> |
| | 35. | <b>Huang W</b>=, Lyman RF=, Carbone MA, Lyman R, Harbison S, Magwire M, Mackay TFC (2016) Spontaneous mutations and the origin and maintenance of quantitative genetic variation. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27213517" taget="_blank"><i>eLife</i> 5:e14625</a> =equal contribution |
| | 34. | Carbone MA, Yamamoto A, <b>Huang W</b>, Lyman RA, Meadors TB, Yamamoto R, Anholt RRH, Mackay TFC (2016) Genetic architecture of natural variation in visual senescence in Drosophila. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27791033" target="_blank"><i>Proc Natl Acad Sci USA</i> 113(43):E6620-E6629</a> |
| | 33. | Hunter CM, <b>Huang W</b>, Mackay TFC, Singh D (2016) The genetic architecture of natural variation in recombination rate in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/27035832" target="_blank"><i>PLoS Genet</i> 12(4):e1005951</a> | 
|<b>2015</b>| 32. | <b>Huang W</b>, Carbone MA, Magwire MM, Peiffer JA, Lyman RF, Stone EA, Anholt RRH, Mackay TFC (2015) Genetic basis of transcriptome diversity in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26483487" target="_blank"><i>Proc Natl Acad Sci USA</i> 112(44):E6010-6019</a> |
| | 31. | Morozova TV=, <b>Huang W</b>=, Pray VA, Whitham T, Anholt RRH, Mackay TFC (2015) Polymorphisms in early neurodevelopmental genes affect natural variation in alcohol sensitivity in adult Drosophila. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26503115" target="blank"><i>BMC Genomics</i> 16(1):865</a> =equal contribution |
| | 30. | Ober U=, <b>Huang W</b>=, Magwire MM, Schlather M, Simianer H, Mackay TFC (2015) Accounting for genetic architecture improves sequence based genomic prediction for a Drosophila fitness trait. <a href="https://www.ncbi.nlm.nih.gov/pubmed/25950439" target="_blank"><i>PLoS ONE</i> 10(5):e0126880</a> =equal contribution |
| | 29. | Dembeck LM, B&ouml;r&ouml;czky K, <b>Huang W</b>, Schal C, Anholt RRH, Mackay TFC (2015) Genetic architecture of natural variation in cuticular hydrocarbon composition in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26568309" target="_blank"><i>eLife</i> 4:e09861</a> |
| | 28. | Dembeck LM, <b>Huang W</b>, Carbone MA, Mackay TFC (2015) Genetic basis of natural variation in body pigmentation in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26554300" target="blank"><i>Fly</i> 9(2):75-81</a> |
| | 27. | Carnes MU, Campbell T, <b>Huang W</b>, Butler DG, Carbone MA, Duncan LH, Harbajan SV, King EM, Peterson KR, Weitzel A, Zhou S, Mackay TFC (2015) The genomic basis of postponed senescence in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26378456" target="_blank"><i>PLoS ONE</i> 10(9):e0138569</a> |
| | 26. | Garlapow ME, <b>Huang W</b>, Yarboro MT, Peterson KR, Mackay TFC (2015) Quantitative genetics of food intake in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26375667" target="_blank"><i>PLoS ONE</i> 10(9):e0138129</a> |
| | 25. | Shorter J, Couch C, <b>Huang W</b>, Carbone, MA, Peiffer J, Anholt RRH, Mackay TFC (2015) Genetic architecture of natural variation in <i>Drosophila melanogaster</i> aggressive behavior. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26100892" target="_blank"><i>Proc Natl Acad Sci USA</i> 112(27):E3555-3563</a> |
| | 24. | Dembeck LM, <b>Huang W</b>, Magwire MM, Lawrence F, Lyman RF, Mackay TFC (2015) Genetic architecture of abdominal pigmentation in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/25933381" target="_blank"><i>PLoS Genet</i> 11(5):e1005163</a> |
| | 23. | Arya GH, Magwire MM, <b>Huang W</b>, Serrano-Negron YL, Mackay TFC, Anholt RRH (2015) The genetic basis for variation in olfactory behavior in <i>Drosophila melanogaster</i>. <a href="https://www.ncbi.nlm.nih.gov/pubmed/25687947" target="_blank"><i>Chem Senses</i> 40:233-243</a> | 
|<b>2014</b>| 22. | <b>Huang W</b>=, Massouras A=, Inoue Y, Peiffer J, R&agrave;mia M, Tarone AM, Turlapati L, Zichner T, Zhu D, Lyman RF, Magwire MM, Blankenburg K, Carbone MA, Chang K, Ellis LL, Fernandez S, Han Y, Highnam G, Hjelmen CE, Jack JR, Javaid M, Jayaseelan J, Kalra D, Lee S, Lewis L, Munidasa M, Ongeri F, Patel S, Perales L, Perez A, Pu L, Rollmann SM, Ruth R, Saada N, Warner C, Williams A, Wu Y, Yamamoto A, Zhang Y, Zhu Y, Anholt RRH, Korbel JO, Mittelman D, Muzny DM, Gibbs RA, Barbadilla A, Johnston JS, Stone EA, Richards S, Deplancke B, Mackay TFC (2014) Natural variation in genome architecture among 205 <i>Drosophila melanogaster</i> Genetic Reference Panel lines. <a href="https://www.ncbi.nlm.nih.gov/pubmed/24714809" target="_blank"><i>Genome Res</i> 24:1193-1208 =equal contribution |
| | 21. | Montgomery SL, Vorojeikina D, <b>Huang W</b>, Mackay TFC, Anholt RRH, Rand MD (2014) Genome-wide association analysis of tolerance to methylmercury toxicity in Drosophila implicates myogenic and neuromuscular developmental pathways. <a href="https://www.ncbi.nlm.nih.gov/pubmed/25360876" target="_blank"><i>PLoS ONE</i> 9(10):e110375</a> |
| | 20. | Ellis LL, <b>Huang W</b>, Quinn AM, Ahuja A, Alfreijd B, Gomez FE, Hjelmen CE, Moore KL, Mackay TFC, Johnston JS, Tarone AM (2014) Intrapopulation genome size variation in <i>D. melanogaster</i> reflects life history variation and plasticity. <a href="https://www.ncbi.nlm.nih.gov/pubmed/25057905" target="_blank"><i>PLoS Genet</i> 10(7):e1004522</a> |
|<b>2013</b>| 19. | Swarup S=, <b>Huang W</b>=, Mackay TFC, Anholt RRH (2013) Analysis of natural variation reveals neurogenetic networks for Drosophila olfactory behavior. <a href="https://www.ncbi.nlm.nih.gov/pubmed/23277560" target="_blank"><i>Proc Natl Acad Sci USA</i> 110(3):1017-1022</a> =equal contribution |
| | 18. | Driver A, <b>Huang W</b>, Kropp J, Pe&ntilde;agaricano F, Khatib H (2013) Knockdown of CDKN1C (p57kip2) and PHLDA2 results in developmental changes in bovine pre-implantation embryos. <a href="https://www.ncbi.nlm.nih.gov/pubmed/23894493" target="_blank"><i>PLoS ONE</i> 8(7):e69490</a> |
|<b>2012</b>| 17. | <b>Huang W</b>, Richards S, Carbone MA, Zhu D, Anholt RRH, Ayroles JF, Duncan L, Jordan KW, Lawrence F, Magwire MM, Warner C, Blankenburg K, Han Y, Javaid M, Jayaseelan J, Jhangiani SN, Muzny D, Ongeri F, Perales L, Wu Y-Q, Zhang Y, Zou X, Stone EA, Gibbs RA, Mackay TFC (2012) Epistasis dominates the genetic architecture of Drosophila quantitative traits. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22949659" target="_blank"><i>Proc Natl Acad Sci USA</i> 109(39):15553-15559</a> |
| | 16. | <b>Huang W</b>, Long N, Khatib H (2012) Genome-wide identification and initial characterization of bovine long non-coding RNAs from EST data. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22497321" target="_blank"><i>Anim Genet</i> 43(6):674-682</a> |
| | 15. | <b>Huang W</b>, Nadeem A, Zhang B, Babar M, Soller M, Khatib H (2012) Characterization and comparison of the leukocyte transcriptomes of three cattle breeds. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22291923" target="_blank"><i>PLoS ONE</i> 7(1):e30244</a> |
| | 14. | <b>Huang W</b>, PenÌagaricano F, Ahmad KR, Lucey JA, Weigel KA, Khatib H (2012) Association between milk protein gene variants and protein composition traits in dairy cattle. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22192223" target="_blank"><i>J Dairy Sci</i> 95:440-449</a> |
| | 13. | Driver A=, Pe&ntilde;agaricano F=, <b>Huang W</b>, Ahmad K, Hackbart K, Wiltbank M, Khatib H (2012) RNA-Seq analysis uncovers transcriptomic variations between morphologically similar in vivo- and in vitro-derived bovine blastocysts. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22452724" target="_blank"><i>BMC Genomics</i> 13:118</a> =equal contribution |
| | 12. | Wu S, Li C, <b>Huang W</b>, Li W, Li R (2012) Alternative splicing regulated by butyrate in bovine epithelial cells. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22720068" target="_blank"><i>PLoS ONE</i> 7(6):e39182</a> |
|<b>2010</b>| 11. | <b>Huang W</b>, Khatib H (2010) Comparison of transcriptomic landscapes of bovine embryos using RNA-Seq. <a href="https://www.ncbi.nlm.nih.gov/pubmed/21167046" target="_blank"><i>BMC Genomics</i> 11:711</a> |
| | 10. | <b>Huang W</b>, Kirkpatrick BW, Rosa GJM, Khatib H (2010) A genome wide association study using selective DNA pooling identifies candidate markers for fertility in Holstein cattle. <a href="https://www.ncbi.nlm.nih.gov/pubmed/20394602" target="_blank"><i>Anim Genet</i> 41(6):570- 578</a> |
| | 9. | <b>Huang W</b>, Yandell BS, Khatib H (2010) Transcriptomic profiling of bovine IVF embryos revealed candidate genes and pathways involved in early embryonic development. <a href="https://www.ncbi.nlm.nih.gov/pubmed/20064253" target="_blank"><i>BMC Genomics</i> 11:23</a> |
| | 8. | Khatib H, Monson RL, <b>Huang W</b>, Khatib R, Schutzkus V, Khateeb H, Parrish JJ (2010) Validation of in vitro fertility genes in a Holstein bull population. <a href="https://www.ncbi.nlm.nih.gov/pubmed/20412940" target="_blank"><i>J Dairy Sci</i> 93(5):2244-2249</a> |
|<b>2009</b> | 7. | <b>Huang W</b>, Mikhail D, Bindrim AC, Khatib H (2009) Interactions of the bovine placental lactogen and prolactin receptor genes are associated with fertility traits in cattle. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22443559" target="_blank"><i>Animal</i> 3(12):1743-1745</a> |
| | 6. | Khatib H, <b>Huang W</b>, Mikhail D, Schutzkus V, Monson RL (2009) Effects of signal transducer and activator of transcription (STAT) genes STAT1 and STAT3 genotypic combinations on fertilization and embryonic survival rates in Holstein cattle. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19923622" target="_blank"><i>J Dairy Sci</i> 92(12):6186-6191</a> |
| | 5. | Wang X, Schutzkus V, <b>Huang W</b>, Rosa GJM, Khatib H (2009) Analysis of segregation distortion and association of the bovine FGF2 with fertilization rate and embryonic survival. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19456315" target="_blank"><i>Anim Genet</i> 40(5):722- 728</a> |
| | 4. | Driver AM, <b>Huang W</b>, Gajic S, Monson RL, Rosa GJM, Khatib H (2009) Effects of the progesterone receptor variants on fertility traits in cattle. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19620692" target="_blank"><i>J Dairy Sci</i> 92(8):4082-4085</a>|
| | 3. | Khatib H, <b>Huang W</b>, Wang X, Tran AH, Bindrim AC, Schutzkus V, Monson RL, Yandell BS (2009) Single gene and gene interaction effects on fertilization and embryonic survival rates in cattle. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19389983" target="_blank"><i>J Dairy Sci</i> 92(5):2238-2247</a> |
| <b>2008</b> | 2. | <b>Huang W</b>, Maltecca C, Khatib H (2008) A proline-to-histidine mutation in POU1F1 is associated with production traits in dairy cattle. <a href="https://www.ncbi.nlm.nih.gov/pubmed/18557974" target="_blank"><i>Anim Genet</i> 39(5):554-557</a>|
| <b>2006</b> | 1. | <b>Huang W</b>=, Dong Z=, Wei H, Ding C, Sun R, Tian Z (2006) Selective elimination of hepatic natural killer T cells with Concanavalin A improves liver regeneration in mice. <a href="http://www.ncbi.nlm.nih.gov/pubmed/16584397" target="_blank"><i>Liver Int</i> 26(3):339-345</a> =equal contribution |

